be/src/exec/pipeline/fragment_executor.cpp

```diff
@@ -89,7 +90,8 @@ Status FragmentExecutor::_prepare_query_ctx(ExecEnv* exec_env, const UnifiedExec
     // duplicate invocations of rpc exec_plan_fragment.
     const auto& params = request.common().params;
     const auto& query_id = params.query_id;
-    const auto& fragment_instance_id = request.fragment_instance_id();
+    const auto& fragment_instance_id = params.fragment_instance_id;
```

Comment:
Must use `request.fragment_instance_id()`, because `fragment_instance_id` only stored in `request._unique_params` for now.